tests/test_exporters.py

```diff
@@ -267,19 +276,19 @@ def test_nested_item(self):
         i3 = TestItem(name=u'buz', age=i2)
 
         self.assertExportResult(i3,
-            '<?xml version="1.0" encoding="utf-8"?>\n'
-            '<items>'
-                '<item>'
-                    '<age>'
-                        '<age>'
-                            '<age>22</age>'
-                            '<name>foo\xc2\xa3hoo</name>'
-                        '</age>'
-                        '<name>bar</name>'
-                    '</age>'
-                    '<name>buz</name>'
-                '</item>'
-            '</items>'
+            u'<?xml version="1.0" ?>\n'
```

Comment:
So, this is just setting up the expectation, the thing is that lxml.etree.tostring complains if you give it a text object with `encoding="utf-8"` (it was needed before because the expected value was defined in bytes).

They're "XML equivalent":

```
$ python
Python 3.4.3 (default, May 27 2015, 20:22:45) 
[GCC 4.8.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import lxml.etree
>>> b = b'<?xml version="1.0" encoding="utf-8"?>\n<items>hello</items>'
>>> u = u'<?xml version="1.0"?>\n<items>hello</items>'
>>> def xmltuple(elem):
...     children = list(elem.iterchildren())
...     if children:
...         return [(child.tag, sorted(xmltuple(child)))
...                 for child in children]
...     else:
...         return [(elem.tag, [(elem.text, ())])]
... 
>>> xmltuple(lxml.etree.fromstring(b)) == xmltuple(lxml.etree.fromstring(u))
True
>>> 
```

So, we could make the expectation bytes and keep the `encoding="utf-8"` and it would be equivalent, what do you think?